lms/djangoapps/verify_student/tests/test_views.py

```diff
@@ -1870,14 +1870,14 @@ def test_reverify_view_can_reverify_expired(self):
         # Allow the student to reverify
         self._assert_can_reverify()
 
-    def test_reverify_view_cannot_reverify_pending(self):
+    def test_reverify_view_can_reverify_pending(self):
```

Comment:
@afeef you are right but its always better to include more details which help understand the code base.